pkg/client/unversioned/scale.go

```diff
@@ -57,3 +58,22 @@ func (c *scales) Get(kind string, name string) (result *expapi.Scale, err error)
 	err = fmt.Errorf("Kind not supported: %s", kind)
 	return
 }
+
+func (c *scales) Update(kind string, scale *expapi.Scale) (result *expapi.Scale, err error) {
```

Comment:
We should add a unittest for scale client: scale_test.go, for both Get and Update. (It can be done in a separate PR).